src/selfplay/tournament.cc

```diff
@@ -127,7 +127,6 @@ void SelfPlayTournament::PopulateOptions(OptionsParser* options) {
                              "multiplexing");
   defaults->Set<bool>(SearchParams::kStickyEndgamesId.GetId(), false);
   defaults->Set<bool>(SearchParams::kLogitQId.GetId(), false);
-  defaults->Set<bool>(SearchParams::kRootHasOwnCpuctParamsId.GetId(), false);
```

Comment:
Also remove the parameter from `params.cc`? As the only reason it existed is actually setting it to false for selfplay.

Reply:
We can't remove it - the server still has to send it until the server enforces a binary where the default is true - Which is what I am doing here.

Reply:
Add a comment so we don't forget?

Reply:
over in params.cc - sure.